benchmarks/run.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -Eexuo pipefail
 
+export DFX_WARNING=-version_check
```

Comment:
I'm not sure we want to suppress this warning. Different versions of dfx can report different instruction counts, so it's important to use the latest version for all benchmarks IMHO.

Would you mind also adding what the benchmarks numbers are before/after this change? Note that #40 needs to be merged before you're able to do so.

Reply:
It's the opposite, there is a newer beta `dfx`, but I'm fine to drop it. #40 was re-implemented in this MR in fact, so completely dropping [this commit](https://github.com/dfinity/stable-structures/pull/43/commits/31f77ce8f514447c8f419ff15d3fac2e3730a90d)... 🤣